src/azure-cli/azure/cli/command_modules/acs/_validators.py

```diff
@@ -73,6 +75,31 @@ def validate_create_parameters(namespace):
         raise CLIError('--dns-prefix has no value')
 
 
+def validate_ip_ranges_on_create(namespace):
+    if (namespace.api_server_authorized_ip_ranges and
+            namespace.load_balancer_sku and
+            namespace.load_balancer_sku.lower() == "basic"):
```

Comment:
Does python SDK has a enum to lb sku? If so, use it instead of a string literal.

Reply:
Good thinking, will check

Reply:
Ok, it looks to me like there is an enum for the standard sku but not basic. https://github.com/Azure/azure-sdk-for-python/blob/1dbf2d60c0de746a84465399354e743b3ac575c7/sdk/aks/azure-mgmt-devspaces/azure/mgmt/devspaces/models/dev_spaces_management_client_enums.py#L26

We could check against the standard sku and assume that if the customer specified standard then they explicitly didn't specify basic but I am hesitant for two reasons:
1. Capitalization, now I need to set the case to be upper case first letter and lower all which requires unneeded code
2. If we introduce a third sku in the future (e.g. premium) then this work around will fail for the new sku 

I prefer to leave it but open to your thoughts if you feel strongly about using the existing enum

Reply:
No problem.